include/certifier_framework.h

```diff
@@ -63,14 +63,15 @@ namespace certifier {
       key_message policy_key_;
 
     public:
-      policy_store(int max_ents);
+      policy_store(unsigned max_ents);
       policy_store();
       ~policy_store();
 
+private:
+      bool add_entry(const string& tag, const string& type, const string& value);
+public:
       bool is_policy_key_valid();
       unsigned get_num_entries();
-      bool add_entry(const string& tag, const string& type, const string& value);
-      int find_entry_by_tag(const string& tag);
```

Comment:
Now that `find_entry_by_tag()` is withdrawn, can we consider simplifying the remaining interface to just:

`find_entry(const string& tag, const string& type)`

The uniqueness is now being enforced by (tag, type), so I feel it's unnecessary to have that longer name.

If ever we find a need to add a new interface with just (tag) or just (type), we can name those methods using the "_by_" sub-term.

Reply:
@jlmucb -- Btw, I should remark here that an interface with both `tag` and `type` of string type is likely to cause errors, when someone mistakenly interchanges the two arguments.

I don't have any clear suggestion how-to address this potential hole, other than that we may want to consider changing `type` to an int or some enumerated value.